qutebrowser/extensions/loader.py

```diff
@@ -180,7 +227,15 @@ def _on_config_changed(changed_name: str) -> None:
                 cfilter = config.change_filter(option)
                 cfilter.validate()
                 if cfilter.check_match(changed_name):
-                    hook()
+                    try:
+                        hook()
+                    except Exception:
+                        log.extensions.exception(
+                            "Failed running config change hook for "
+                            "item {} in extension: {}".format(
+                                changed_name, hook.__code__.co_filename,
```

Comment:
This seems a bit hacky for getting the name - why not add that information to `ModuleInfo` or so?